easybuild/easyconfigs/p/pycocotools/pycocotools-2.0.0-foss-2019b-Python-3.7.4.eb

```diff
@@ -0,0 +1,25 @@
+easyblock = 'PythonPackage'
+
+name = 'pycocotools'
+version = '2.0.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/cocodataset/cocoapi'
+description = "Tools for working with the MSCOCO dataset"
+
+toolchain = {'name': 'foss', 'version': '2019b'}
```

Comment:
@hajgato This looks fine. The only question is whether it needs MPI/BLAS etc., or whether it should live at a lower toolchain e.g. `GCCcore/8.3.0`.

Reply:
Well, it needs SciPy, and we do not have that without MPI/BLAS as far as I know.